src/switch_core_media.c

```diff
@@ -17699,16 +17699,27 @@ SWITCH_DECLARE(switch_status_t) switch_core_session_write_frame(switch_core_sess
 			session->enc_write_frame.datalen = session->enc_write_frame.buflen;
 			session->write_codec->cur_frame = frame;
 			frame->codec->cur_frame = frame;
-			switch_assert(enc_frame->datalen <= SWITCH_RECOMMENDED_BUFFER_SIZE);
-			switch_assert(session->enc_write_frame.datalen <= SWITCH_RECOMMENDED_BUFFER_SIZE);
+			if (enc_frame->datalen > SWITCH_RECOMMENDED_BUFFER_SIZE || session->enc_write_frame.datalen > SWITCH_RECOMMENDED_BUFFER_SIZE) {
+				switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_ERROR, "Encoded write frame greater than %d\n",
```

Comment:
Include the size of both enc_frame and enc_write_frame datalen in the log.

Reply:
Done